source/Calamari.Shared/Integration/Packages/Download/Scripts/DockerLogin.ps1

```diff
@@ -0,0 +1,51 @@
+function IsDockerAvailable() {
```

Comment:
Do these new scripts automatically get included in any built version of Calamari? Just want to be sure we don't need to explicitly add them to the project file or something along those lines.

Reply:
Yep, these scripts are auto included because previously the scripts under this folder are added as embedded resources using the wildcard.